README.md

```diff
@@ -247,4 +247,24 @@ dependencies {
 
 ### Usage
 
-// TODO
\ No newline at end of file
+Create new `Lyrics Manager`:
+```java
+var lyricsManager = new LyricsManager();
+```
```

Comment:
also mention the need to register sources here

Reply:
A quick

> Remember to register the sources

or a whole example?

Reply:
or a comment insde of the `java block`?

Reply:
a comment with some pseudo code like this should be fine
```suggestion
lyricsManager.registerLyricsManager(sourcer)
```